test/e2e/storage/persistent_volumes.go

```diff
@@ -319,6 +325,260 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 		})
 	})
 
+	ginkgo.Describe("CSI Conformance", func() {
```

Comment:
Is it target for the Conformance test?

Reply:
Yes, the target for the test is Conformance. Issue https://github.com/kubernetes/kubernetes/issues/119694 has more details of the storage endpoints we hope to add to Conformance in the near future.